tests/testthat/test-multitables.R

```diff
@@ -228,6 +230,12 @@ with_test_authentication({
         expect_identical(names(refresh(multitables(ds))), "Yet another name")
     })
 
+    test_that("Can copy the multitable to a new multitable", {
+        m <- importMultitable(name='copied_multitable', data=ds, multitable=mult)
+        expect_identical(name(m), "copied_multitable")
+        is.public(multitables(ds))[2] <- TRUE
```

Comment:
For the integration test, (1) no need to make the multitable public on L236 (rogue copy/paste, I'm guessing); (2) it should also assert something about the contents of the multitable that has been copied to show that the copy was correct. If Multitable objects had a nice print method, that would be easy, but I don't see that they do. Perhaps @jonkeane can help with that.